src/main/com/aca/homework/week10/enums/list/PhoneModel.java

```diff
@@ -0,0 +1,28 @@
+package com.aca.homework.week10.enums.list;
+
+import java.util.Collections;
+import java.util.List;
+
+public enum PhoneModel {
+
+    NOKIA_1100 {
+        @Override
+        public List<String> getCameraDescriptionList() {
+            return Collections.emptyList();
+        }
+    },
+    NOKIA_3110 {
+        @Override
+        public List<String> getCameraDescriptionList() {
+            return Collections.singletonList("back camera");
+        }
+    },
+    GALAXY_S2 {
+        @Override
+        public List<String> getCameraDescriptionList() {
+            return List.of("back camera", "front camera");
```

Comment:
every time a user invoked getCameraDescriptionList() the code will create a new list object. you can set it up once and use it next times